test/resources/shares.js

```diff
@@ -0,0 +1,201 @@
+      expect(req.res.body).to.be.an.array()
+      expect(req.res.body).to.have.length(16)
+      expect(req.res.body.map(item => item.title)).to.contain(titles)
+    }))
+
+    lab.test('deve retornar até as 5 primeiras linhas do banco de dados', co.wrap(function * () {
+      const options = { firstUserSharesQty: 4, secondUserSharesQty: 2 }
+
+      yield fixtures.users.insertMultiple()
+      yield fixtures.shares.insertMultiple(options)
+
+      const baseTitle = 'Veja como utilizar o V8 da melhor maneira - Parte '
+      const req = yield request(server)
+      .get(`${ENDPOINT}?limit=5&page=1`)
+
+      expect(req.statusCode).to.equal(200)
+      expect(req.res.body).to.be.an.array()
+      expect(req.res.body).to.have.length(5)
+      expect(req.res.body.map(item => item.title)).to.contain([
+        baseTitle + '1',
+        baseTitle + '2',
+        baseTitle + '3',
+        baseTitle + '4',
+        'Introdução a NodeJS - Parte 1'
+      ])
+    }))
+
+    lab.test('deve pular as 5 primeiras linhas do banco de dados e retornar a seguinte', co.wrap(function * () {
+      const options = { firstUserSharesQty: 4, secondUserSharesQty: 2 }
+
+      yield fixtures.users.insertMultiple()
+      yield fixtures.shares.insertMultiple(options)
+
+      const req = yield request(server)
+      .get(`${ENDPOINT}?limit=5&page=2`)
+
+      expect(req.statusCode).to.equal(200)
+      expect(req.res.body).to.be.an.array()
+      expect(req.res.body).to.have.length(1)
+      expect(req.res.body.map(item => item.title)).to.contain([
+        'Introdução a NodeJS - Parte 2'
+      ])
+    }))
+  })
+
+  lab.describe(`GET ${ENDPOINT}/:id`, () => {
+    lab.test('deve retornar a linha selecionada', co.wrap(function * () {
+      const id = uuid.v4()
+
+      yield fixtures.users.insertMultiple()
+      yield fixtures.shares.insertMultipleWithId([id])
+
+      const req = yield request(server)
+      .get(`${ENDPOINT}/${id}`)
+
+      expect(req.statusCode).to.equal(200)
+      expect(req.res.body).to.be.an.object()
+      expect(req.res.body).to.contain({
+        id: id,
+        user_id: firstUserId,
+        title: `Veja como utilizar o V8 da melhor maneira - Parte 1`,
+        thumbnail: 'https://google.com.br/logo',
+        link: 'https://google.com.br'
+      })
+    }))
+
+    lab.test('não deve encontrar linhas', co.wrap(function * () {
+      const id = uuid.v4()
+      const otherId = uuid.v4()
+
+      yield fixtures.users.insertMultiple()
+      yield fixtures.shares.insertMultipleWithId([id])
+
+      const req = yield request(server)
+      .get(`${ENDPOINT}/${otherId}`)
+
+      expect(req.statusCode).to.equal(404)
+    }))
+  })
+
+  lab.describe(`DELETE ${ENDPOINT}/:id`, () => {
+    lab.test('deve remover do banco de dados a linha selecionada', co.wrap(function * () {
+      const id = uuid.v4()
+
+      yield fixtures.users.insertMultiple()
+      yield fixtures.shares.insertMultipleWithId([id])
+
+      const req = yield request(server)
```

Comment:
@arun1595 well spotted man. I'm switching the first req to a simple yield, it should fix this issue